packages/create-sitecore-jss/src/templates/nextjs-sxa/src/styles/globals.css

```diff
@@ -0,0 +1,26 @@
+html,
```

Comment:
Can it be defined in `styles.css`? Which you already have.

Reply:
I checked this file - there isn't use anywhere - removed